integration_tests/tests/v2Manage/outOfServiceBeds.cy.ts

```diff
@@ -113,33 +114,71 @@ context('OutOfServiceBeds', () => {
     page.shouldShowDateConflictErrorMessages(conflictingBooking, 'booking')
   })
 
-  it('should show a out of service bed', () => {
-    // Given I am signed in as a future manager
-    signIn(['future_manager'])
+  describe('for a new out of service bed with all nullable fields present in the initial OoS bed revision', () => {
+    it('should show a out of service bed', () => {
+      // Given I am signed in as a future manager
+      signIn(['future_manager'])
 
-    // And I have created a out of service bed
-    const bed = { name: 'abc', id: '123' }
-    const premises = premisesFactory.build()
-    const outOfServiceBed = outOfServiceBedFactory.build({ bed })
-    const bedDetail = bedDetailFactory.build({ id: bed.id })
+      // And I have created a out of service bed
+      const bed = { name: 'abc', id: '123' }
+      const premises = premisesFactory.build()
+      const outOfServiceBed = outOfServiceBedFactory.build({ bed })
+      const bedDetail = bedDetailFactory.build({ id: bed.id })
 
-    cy.task('stubOutOfServiceBed', { premisesId: premises.id, outOfServiceBed })
-    cy.task('stubBed', { premisesId: premises.id, bedDetail })
+      cy.task('stubOutOfServiceBed', { premisesId: premises.id, outOfServiceBed })
+      cy.task('stubBed', { premisesId: premises.id, bedDetail })
 
-    // And I visit that out of service bed's show page
-    const page = OutOfServiceBedShowPage.visit(premises.id, outOfServiceBed)
+      // And I visit that out of service bed's show page
+      const page = OutOfServiceBedShowPage.visit(premises.id, outOfServiceBed)
 
-    // Then I should see the latest details of that out of service bed
-    page.shouldShowOutOfServiceBedDetail()
+      // Then I should see the latest details of that out of service bed
+      page.shouldShowOutOfServiceBedDetail()
 
-    // And I should see the bed characteristics
-    page.shouldShowCharacteristics(bedDetail)
+      // And I should see the bed characteristics
+      page.shouldShowCharacteristics(bedDetail)
 
-    // When I click the 'Timeline' tab
-    page.clickTab('Timeline')
+      // When I click the 'Timeline' tab
+      page.clickTab('Timeline')
 
-    // Then I should see the timeline of that out of service bed's revision
-    page.shouldShowTimeline()
+      // Then I should see the timeline of that out of service bed's revision
+      page.shouldShowTimeline()
+    })
+  })
+
+  describe('for a legacy "lost bed" with all nullable fields not present in the initial OoS bed revision', () => {
```

Comment:
is it worth including the fact that this may be the case for *migrated* records e.g 

```
'for a legacy "lost bed" records migrated with null fields'
```